kasa/klaptransport.py

```diff
@@ -432,6 +435,8 @@ def __init__(self, local_seed, remote_seed, user_hash):
         self.user_hash = user_hash
         self._key = self._key_derive(local_seed, remote_seed, user_hash)
         (self._iv, self._seq) = self._iv_derive(local_seed, remote_seed, user_hash)
+        self._aes = algorithms.AES(self._key)
+        self._generate_cipher()
```

Comment:
This seems like a good optimisation as it's no longer re-creating the same cipher in `decrypt` but does it need to create it in `__init__` or could that also be avoided?  N.B. `decrypt` is always called after `encrypt`.

Reply:
I removed it from init.